frontend/fingertips-frontend/components/pages/search/results/searchResultsActions.test.ts

```diff
@@ -8,11 +8,14 @@ const redirectMock = jest.mocked(redirect);
 const initialState: SearchResultState = {
   indicatorsSelected: [],
 };
+let formData: FormData;
 
 describe('Search Results Actions', () => {
   describe('viewCharts', () => {
+    beforeEach(() => {
+      formData = new FormData();
+    });
```

Comment:
I think this one I would prefer to how it was. I seeing explicitly within the test that I am starting out with a blank formData class.